src/app/my-file/components/my-file/my-file.component.html

```diff
@@ -3,7 +3,7 @@
         <table class="user-info-table1">
             <tr>
                 <td class="user-info-categories">Angestellt seit:</td>
-                <td class="user-info-values">12.11.2020</td>
+                <td class="user-info-values">{{signInDate}}</td>
```

Comment:
You could also use `user.sign_up_timestamp` in combination with the Angular date pipe (https://angular.io/api/common/DatePipe)